src/tests/ftest/soak/utils.py

```diff
@@ -198,22 +196,16 @@ def get_remote_dir(self, source_dir, dest_dir, host_list, shared_dir=None,
         # copy the local logs and the logs in the shared dir to avocado dir
         for directory in [source_dir, shared_dir]:
             command = "/usr/bin/cp -R -p {0}/ \'{1}\'".format(directory, dest_dir)
-            try:
-                run_command(command, timeout=30)
-            except DaosTestError as error:
-                raise SoakTestError("<<FAILED: job logs failed to copy>>") from error
+            run_command(command, timeout=30, raise_exception=False)
```

Comment:
We should reinstate the SoakTestError raising for non-tearDown calling and instead use a try .. except .. in the pre_tear_down() method to capture any errors and perform the appropriate action (log a message, or add the error to the list of errors to be reported at the end of tearDown, etc.)